app/src/app/[lng]/data/[step]/page.tsx

```diff
@@ -54,6 +55,49 @@ function getMailURI(locode?: string, sector?: string, year?: number): string {
   return `mailto://info@openearth.org,greta@openearth.org?subject=Missing third party data sources&body=City: ${locode}%0ASector: ${sector}%0AYear: ${year}`;
```

Comment:
Could be also nice to have these emails in our env variables in future.

Reply:
That's a good point :+1:
I'll add a variable for that.

Reply:
There is now `process.env.SUPPORT_EMAILS`, but I added the current emails as a default value.